buildtools/packer/aws-ubuntu.pkr.hcl

```diff
@@ -57,8 +57,11 @@ build {
   provisioner "shell" {
     inline = [
       "whoami",
+      "sudo apt update",
+      "sudo apt install -y \\",
+      "  libudev-dev",
```

Comment:
should we include this as part of the `dev_setup.sh`? There seems to be function to install this already, but it just seems to be not called properly. can we fix that?

https://github.com/aptos-labs/aptos-core/blob/44ab92360904589e08f379b35ef557382103e2f7/scripts/dev_setup.sh#L1106

Reply:
Updated `dev_setup.sh` to install this package